### ISSUE TITLE
[BUG]: Cosmos-UI Does not work in Safari - macOS

### ISSUE THREAD

**Issue by yeradis** [azukaar/Cosmos-Server#209]
### What happened?

First time install and UI does not load at http://localhost/cosmos-ui

### What should have happened?

Show Cosmos Setup UI

### How to reproduce the bug?

- On macOS follow the instructions at https://github.com/azukaar/Cosmos-Server?tab=readme-ov-file#installation
- Go to http://your-server-ip

### Relevant log output

```shell
Safari Console output:


[Error] Error parsing client-infos cookie:
TypeError: undefined is not an object (evaluating 'e["client-infos"].split')
CIe — index-50aebb50.js:378:5197
MD — index-50aebb50.js:378:5435
Xw — index-50aebb50.js:38:19540
e4 — index-50aebb50.js:40:44058
QM — index-50aebb50.js:40:39790
S7 — index-50aebb50.js:40:39718
_h — index-50aebb50.js:40:39571
YM — index-50aebb50.js:40:34650
C — index-50aebb50.js:25:1566
H — index-50aebb50.js:25:1927
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] Error parsing client-infos cookie: – TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
TypeError: undefined is not an object (evaluating 'e["client-infos"].split') — index-50aebb50.js:378:5197
	CIe (index-50aebb50.js:378:5264)
	MD (index-50aebb50.js:378:5435)
	Xw (index-50aebb50.js:38:19540)
	e4 (index-50aebb50.js:40:44058)
	QM (index-50aebb50.js:40:39790)
	S7 (index-50aebb50.js:40:39718)
	_h (index-50aebb50.js:40:39571)
	YM (index-50aebb50.js:40:34650)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] TypeError: undefined is not an object (evaluating 'n.filter') — index-50aebb50.js:384:17470
	Q1 (index-50aebb50.js:40:167)
	(anonymous function) (index-50aebb50.js:40:390)
	i$ (index-50aebb50.js:38:9891)
	S$ (index-50aebb50.js:40:32063)
	KM (index-50aebb50.js:40:31482)
	b7 (index-50aebb50.js:40:31032)
	E7 (index-50aebb50.js:40:40920)
	ma (index-50aebb50.js:40:40278)
	YM (index-50aebb50.js:40:35190)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
[Error] TypeError: undefined is not an object (evaluating 'n.filter')
	E7 (index-50aebb50.js:40:41204)
	ma (index-50aebb50.js:40:40278)
	YM (index-50aebb50.js:40:35190)
	C (index-50aebb50.js:25:1566)
	H (index-50aebb50.js:25:1927)
```
```


### Other details

_No response_

### System details

- OS: macOS Sonoma 14.2.1
- Browser: Safari
- Version 17.2.1 (19617.1.17.11.12)

**Comment by azukaar** [comment]
I cannot reproduce this. I might have fixed it in 0.15 without knowing, could one of you try the latest beta version (with :latest-unstable) and let me know if it makes a difference?

**Comment by azukaar** [comment]
sorry this doesn't really help atm, did you try with 0.15?

**Comment by azukaar** [comment]
> Cosmos Server isn't a real solution 

Alright thanks for telling me ahead of this block of text that it was not worth answering

**Comment by azukaar** [comment]
Alright dude, thanks for the ChatGPT response. May be it's time for you to take some constructive feedback too: re-evaluate how you interact with people who provide you with hard worked on software for free. You want me to priorities Safari for you with that ass attitude? ... No way!

I will now block you from this repository to prevent this legitimate ticket to further degrade into a ridiculous skirmish.

As I said, **I cannot reproduce the issue**. This is how far it goes until someone helps me reproduce it, or confirm that it is fixed in 0.15

**Comment by azukaar** [comment]
For anyone struggling to access Cosmos altogether on MacOS: make sure you run without the host mode (see https://cosmos-cloud.io/doc/1%20index/) as it will not work on MacOS
